### PR TITLE
Fitness App UI

### PR DESCRIPTION
**Pull request by Redvey**
#Hacktoberfest 2023
I have made the UI of the Fitness app according to the issue statement, and also updated the Readme File about my PR.
It consists of 3 pages.

https://github.com/chandansgowda/flutter_ui_kit/assets/128305006/2c5717e1-fa42-403f-bd8f-f2d084f7a485

For #reference
image

These are the 3 pages given
Screenshot 2023-10-24 at 10 11 28 PM
Screenshot 2023-10-24 at 10 11 18 PM
Screenshot 2023-10-24 at 10 11 24 PM

### CODE REVIEW

**Comment by chandansgowda** [comment]
@Redvey Please resolve conflicts and also add a demo video.

**Comment by Redvey** [comment]
@chandansgowda  Hey, I resolved the conflicts and also added a demo video.

**Comment by Redvey** [comment]
@chandansgowda Hello there, any update?

File: flutter_ui_kit/lib/fitness_app/home_screen_widgets/Categories scroll.dart
Comment (chandansgowda) [inline flutter_ui_kit/lib/fitness_app/home_screen_widgets/Categories scroll.dart:1]: Rename the file and use snake case

File: flutter_ui_kit/lib/fitness_app/consts/Pallete.dart
Comment (chandansgowda) [inline flutter_ui_kit/lib/fitness_app/consts/Pallete.dart:1]: lets use snake case for file name

**Review by chandansgowda — COMMENTED** [review]
Great job till here @Redvey 
Minor changes have to be done.

File: flutter_ui_kit/lib/fitness_app/consts/Pallete.dart
Comment (Redvey) [reply flutter_ui_kit/lib/fitness_app/consts/Pallete.dart:1]: @chandansgowda Hello, made the changes as said above.

**Comment by Redvey** [comment]
@chandansgowda Hello there, my pull request (PR) is currently pending. If there aren't any further changes required, please review and approve it for merging. As you might be aware, Hacktoberfest is drawing to a close soon, on October 31st. Thank you so much for your help.

**Pull request merged**